lib/modules/hosts/wikipedia.js

```diff
@@ -56,7 +56,12 @@ export default new Host('wikipedia', {
 		// Remove unwanted sections
 		$wikiData.find('.metadata, .hatnote, .mw-editsection, .reference, .references').remove();
 		// Make all relative links direct
-		$wikiData.find('a[href^="/"]').attr('href', (i, old) => `https://${language}.wikipedia.org${old}`);
+		$wikiData.find('a')
```

Comment:
The [URL constructor](https://developer.mozilla.org/en/docs/Web/API/URL/URL) is likely what we need here, e.g. ```new URL(old, `https://${language}.wikipedia.org/wiki/${article}`).href```; it can resolve any url relative to a base URL